cogs/stars.py

```diff
@@ -3,15 +3,16 @@
 from nextcord.ext.tasks import loop
 
 
+STARS_CHANNEL_ID = 884441198520045570
+
 class GitHubStars(commands.Cog):
     """Run a task loop to update a channel with the current stars for nextcord/nextcord and nextcord/nextcord-v3"""
 
-    STARS_CHANNEL_ID = 884441198520045570
-
     def __init__(self, bot: commands.Bot):
-        self.__bot = bot
+        self.bot = bot
```

Comment:
If you change this, you'd also have to change everywhere that uses self.__bot

It's a better practice to make things private (double underscore) if they shouldn't be used outside the class itself, so I don't think this should be changed.

Since everywhere else in the code it's not following that practice, you could change it if you really want to, I guess.

In a bot like this, I don't think it's extremely important for modules by different authors to be consistent with each other. Consistency within a module is much more important than consistency across different modules in my opinion..